harpy/align.py

```
@@ -4,13 +4,13 @@
 import yaml
 import shutil
-from pathlib import Path
 import rich_click as click
 from ._conda import create_conda_recipes
-from ._misc import fetch_report, fetch_rule, snakemake_log
+from ._misc import fetch_report, fetch_rule, snakemake_log, write_snakemake_config, write_workflow_config
 from ._cli_types_generic import convert_to_int, ContigList, InputFile, HPCProfile, SnakemakeParams
 from ._cli_types_params import BwaParams, EmaParams, StrobeAlignParams
-from ._launch import launch_snakemake, SNAKEMAKE_CMD
+from ._launch import launch_snakemake
+from ._misc import filepath
```

Ruff (0.8.2)

`._misc.filepath` imported but unused

Remove unused import: `._misc.filepath`

(F401)